### PR TITLE
use strict_cookie mthod for cookie storing, remove jquery filedownloa…

### LINKED ISSUE #77: Firefox complains about cookie same site attribute
Apparently, there is a problem with our cookies and their same site attribute. In the future, these cookies will be declined.

### PR DESCRIPTION
**Pull request by fosterfarrell9**
This should fix #77. In the process, the jquery_filedownload-rails gem was removed (it triggered a cookie warning and is not really necessary).

### CODE REVIEW

File: app/controllers/courses_controller.rb [hunk 1]
```diff
@@ -9,7 +9,6 @@ class CoursesController < ApplicationController
 
   def edit
     I18n.locale = @course.locale || I18n.default_locale
-    cookies[:edited_course] = params[:id]
```
Comment (henrixapp) [inline app/controllers/courses_controller.rb:12]: Why is this line removed?

File: Gemfile.lock [hunk 2]
```diff
@@ -246,15 +246,12 @@ GEM
       nokogiri (>= 1.6)
     jbuilder (2.10.0)
       activesupport (>= 5.0.0)
-    jquery-datetimepicker-rails (2.4.1.0)
```
Comment (henrixapp) [inline Gemfile.lock:249]: Was this removed earlier?

**Review by henrixapp — COMMENTED** [review]
We have to discuss the use of `x_sendfile`.

File: app/controllers/courses_controller.rb (hunk 1 above)
Comment (fosterfarrell9) [reply app/controllers/courses_controller.rb:12]: I did not see any place in the codebase where this cookie is actually used (anymore).

File: Gemfile.lock (hunk 2 above)
Comment (fosterfarrell9) [reply Gemfile.lock:249]: That is just an artefact of me playing around with that gem.
